### ISSUE TITLE
XUV Snap sub

### ISSUE THREAD

**Issue by Anki-code1** [Anki-code1/sushiwill25#1]
is:pr author:@me state:open -is:draft review:approved -review:changes_requested -status:failure -is:queued archived:false sort:updated-desc
